test/db/analysis/x86_32

```diff
@@ -1207,6 +1207,7 @@ EOF
 EXPECT=<<EOF
 jmp 0xf0070
 EOF
+BROKEN=1
```

Comment:
And this one is still broken

Reply:
those 2 i don't even know if they are correct in the first place.

Reply:
@wargio I created this test, it's the same issue as long time ago: https://github.com/capstone-engine/capstone/issues/111